mak/COPY

```diff
@@ -93,6 +108,7 @@ COPY=\
 	$(IMPDIR)\core\sys\osx\mach\thread_act.d \
 	\
 	$(IMPDIR)\core\sys\osx\sys\cdefs.d \
+	$(IMPDIR)\core\sys\osx\sys\event.d \
```

Comment:
I meant just remove it from `mak/COPY`, as it never shipped with an official dmd release and now there's the darwin equivalent.  As for removing the file, I just checked and the header _was_ shipped with ldc, so probably best to treat it like every other `core.sys.osx` header you moved over to `core.sys.darwin`, ie add a doc comment saying it will be deprecated or whatever we decide to do with those.